addon-test-support/@ember/test-helpers/dom/-is-focusable.ts

```diff
@@ -1,14 +1,12 @@
 import isFormControl from './-is-form-control';
-import { isDocument } from './-target';
+import { isDocument, isContentEditable } from './-target';
 
 const FOCUSABLE_TAGS = ['A'];
 
 type FocusableElement = HTMLAnchorElement;
 
 // eslint-disable-next-line require-jsdoc
-function isFocusableElement(
-  element: HTMLElement | SVGElement | Element
-): element is FocusableElement {
+function isFocusableElement(element: Element): element is FocusableElement {
```

Comment:
why did you change the types here?

Reply:
I believe this was my attempt to simlify a bit. Both `HTMLElement` are `SVGElement` derrive from `Element`, so it does seem redundant to specify each.
Also, implementation wise it just checks if it's not an `A` tag!